vcd/resource_vcd_network_routed.go

```diff
@@ -279,14 +280,11 @@ func resourceVcdNetworkDelete(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("error finding network: %#v", err)
 	}
 
-	err = retryCall(vcdClient.MaxRetryTimeout, func() *resource.RetryError {
-		task, err := network.Delete()
-		if err != nil {
-			return resource.RetryableError(
-				fmt.Errorf("error Deleting Network: %#v", err))
-		}
-		return resource.RetryableError(task.WaitTaskCompletion())
-	})
+	task, err := network.Delete()
+	if err != nil {
+		return fmt.Errorf("error Deleting Network: %#v", err)
```

Comment:
""error Deleting Network" -> ""error deleting network"

Reply:
Fixed

Reply:
Still not fixed. `Network` -> `network`

Reply:
Fixed